qa/suites/upgrade/reef-x/parallel/1-tasks.yaml

```diff
@@ -1,33 +1,33 @@
 tasks:
 - install:
-    branch: pacific
+    branch: reef
     exclude_packages:
       - ceph-volume
 - print: "**** done install task..."
-- print: "**** done start installing pacific cephadm ..."
+- print: "**** done start installing reef cephadm ..."
 - cephadm:
-    image: quay.ceph.io/ceph-ci/ceph:pacific
-    cephadm_branch: pacific
+    image: quay.ceph.io/ceph-ci/ceph:reef
+    cephadm_branch: reef
     cephadm_git_url: https://github.com/ceph/ceph
     conf:
       osd:
         #set config option for which cls modules are allowed to be loaded / used
         osd_class_load_list: "*"
         osd_class_default_list: "*"
-- print: "**** done end installing pacific cephadm ..."
+- print: "**** done end installing reef cephadm ..."
 
 - print: "**** done start cephadm.shell ceph config set mgr..."
 - cephadm.shell:
     mon.a:
       - ceph config set mgr mgr/cephadm/use_repo_digest true --force
 - print: "**** done cephadm.shell ceph config set mgr..."
 
-- print: "**** done start telemetry pacific..."
+- print: "**** done start telemetry reef..."
 - workunit:
     clients:
       client.0:
-        - test_telemetry_pacific.sh
-- print: "**** done end telemetry pacific..."
+        - test_telemetry_reef.sh
```

Comment:
Where would this workunit come from?  AFAICS this PR adds only `test_telemetry_reef_x.sh`.

Reply:
Good catch! There is also the need for `test_telemetry_reef.sh`. Adding.